extensions/interactions/FractionInput/directives/oppia-interactive-fraction-input.directive.ts

```diff
@@ -17,6 +17,7 @@
  */
 
 require('domain/objects/FractionObjectFactory.ts');
+require('domain/objects/objects-domain.constants.ajs.ts');
```

Comment:
Please move the constant requires into a separate section that is after this one and is separated by empty line.

Reply:
Done